pandas/core/frame.py

```diff
@@ -6708,6 +6711,17 @@ def unstack(self, level=-1, fill_value=None):
     1  b         B          3
     2  c         B          5
 
+    Original index values can be kept around:
+
+    >>> %(caller)sid_vars=['A'], value_vars=['B', 'C'], index=True)
```

Comment:
not sure what u r doing here

Reply:
Adding an example to the docstring. Passes docstring verification.